Polyclinic/Polyclinic.Api/Controllers/AppointmentsController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Polyclinic.Contracts.Appointments;
+using Polyclinic.Contracts.Doctors;
+using Polyclinic.Contracts.Patients;
+using System.ComponentModel.DataAnnotations;
+
+namespace Polyclinic.Api.Controllers;
+
+/// <summary>
+/// Controller for managing medical appointments
+/// </summary>
+public class AppointmentsController(IAppointmentService crudService, ILogger<AppointmentsController> logger)
+    : CrudControllerBase<AppointmentDto, AppointmentCreateUpdateDto, int>(crudService, logger)
+{
+    /// <summary>
+    /// Retrieves patient information for a specific appointment asynchronously
+    /// </summary>
+    /// <param name="id">Appointment identifier</param>
+    /// <returns>Patient details for the specified appointment</returns>
+    [HttpGet("{id}/patient")]
+    [ProducesResponseType(200)]
+    [ProducesResponseType(404)]
+    [ProducesResponseType(400)]
+    [ProducesResponseType(500)]
+    public async Task<ActionResult<PatientDto>> GetAppointmentPatientAsync(
+        [Range(1, int.MaxValue, ErrorMessage = "Appointment ID must be positive")] int id)
+    {
+        try
+        {
+            var res = await crudService.GetAppointmentPatientAsync(id);
+            return res != null ? Ok(res) : NotFound();
+        }
+        catch (ArgumentException ex)
+        {
+            logger.LogWarning(ex, "Validation error in GetAppointmentPatientAsync");
+            return BadRequest(new { Message = ex.Message });
```

Comment:
Упростить. Имя указывать не нужно так как оно совпадает. Да и в целом, почему бы не вернуть просто текст ошибки
```suggestion
            return BadRequest(ex.Message);
```
Аналогично в других местах.

Reply:
Можете конечно же оставить и вариант с объектом, но уберите `Message = `. Оба варианта имеют место быть. у нас тут простые случаи донести сообщение об ошибки клиенту.